checker/src/org/checkerframework/checker/upperbound/messages.properties

```diff
@@ -1,2 +1,3 @@
 ### Error messages for the Upper Bound Checker
-array.access.unsafe.high=Potentially unsafe array access: the index could be larger than the array's bound\nfound   : %s\nrequired: @LTLengthOf("%s") int
+array.access.unsafe.high=Potentially unsafe array access: the index could be larger than the array's bound\nfound   : %s\nrequired: @LTLengthOf("%s") int\nor: @LTLengthOf("%s")
```

Comment:
Oh, i should have removed that, the second was the viewpoint adapted version, so they were different if viewpoint stuff wasn't working.